### ISSUE TITLE
No availaible OAuth method with firefox

### ISSUE THREAD

**Issue by PLMbugz** [BuggleInc/webPLM#88]
With that navigator (version 44, debian testing), I get a page with the text but no icon. It works fine in chromium.

Thanks, Mt.

**Comment by MatthieuNICOLAS** [comment]
To which page were you referring? https://plm.telecomnancy.univ-lorraine.fr/#/ui/signIn ?
I could not reproduce your issue, neither with Firefox 45.0 nor Firefox Developer Edition 47.0a2

**Comment by mquinson** [comment]
Yes, this is the page I am referring to. If it works with any other versions of FF, then I guess we can live with it. We could also rephrase it to offer directly the login/password of our own OAuth provider, and add the buttons to the external services below. Something similar to what the Khan Academy proposes:
![capture-auth-khan](https://cloud.githubusercontent.com/assets/624847/14024736/fbc676d0-f1ea-11e5-8a28-722966de8ec8.png)

**Comment by mquinson** [comment]
For the record, the problem was due to my adblocker, which cut the script : https://plm.telecomnancy.univ-lorraine.fr/assets/app/components/sign-in/social-buttons.directive.js

The solution would probably to rename that script, or to add a hint at the bottom advising to people not seeing anything to turn their ad blocker off.

**Comment by mquinson** [comment]
This is now fixed, thanks for that.
